frontend/src/viewer/components/2d/path-element.ts

```diff
@@ -319,6 +319,35 @@ export class PathElement extends connect(store)(LitElement) {
     return `http://xctrack.org/xcplanner?${params}`;
   }
 
+  private encodeNumber(num: number): string {
```

Comment:
What do you think of moving those 2 methods to `flyxc/frontend/src/viewer/logic/task.ts` ?

Reply:
I wasn't able to run FlyXc locally so this isn't end-to-end tested by me. Thus I'd rather avoid making more complicated changes. Feel free to move it there in a follow-up commit.

Reply:
Someone (me ?) should write a doc on running this locally

Reply:
The biggest trouble seems to be getting all the keys in `keys.ts.dist`.